docs/src/prototypes/mentions/MentionsEditor.tsx

```diff
@@ -82,8 +82,9 @@ const MentionsEditor: React.FunctionComponent<
     dispatch({ type: 'RESET_UPDATE_FLAG' })
   }, [state.shouldUpdate])
 
-  const handleEditorKeyUp = (e: React.KeyboardEvent) => {
-    if (!state.open && e.shiftKey && keyboardKey.getCode(e) === keyboardKey.AtSign) {
+  const handleEditorKeyChange = (e: any) => {
+    const editorContent = e.target.textContent
+    if (!state.open && editorContent[editorContent.length - 1] === '@') {
```

Comment:
it would be better to check which character is at the cursor position. see for example https://www.techcycle.me/post/reactjs-track-cursor-in-textarea on how to find the selection start/end